commons/ava_testsuite/staking_gecko_network_tests.go

```diff
@@ -196,14 +234,16 @@ func (test FiveNodeStakingNetworkGetValidatorsTest) GetTimeout() time.Duration {
 // TODO TODO TODO Rename this
 func getFiveNodeStakingLoader() (testsuite.TestNetworkLoader, error) {
 	serviceConfigs := map[int]ava_networks.TestGeckoNetworkServiceConfig{
-		NODE_CONFIG_ID: *ava_networks.NewTestGeckoNetworkServiceConfig(true, ava_services.LOG_LEVEL_DEBUG),
+		NODE_CONFIG_ID:           *ava_networks.NewTestGeckoNetworkServiceConfig(true, ava_services.LOG_LEVEL_DEBUG),
+		DELEGATOR_NODE_CONFIG_ID: *ava_networks.NewTestGeckoNetworkServiceConfig(true, ava_services.LOG_LEVEL_DEBUG),
 	}
 	return ava_networks.NewTestGeckoNetworkLoader(
 		ava_services.LOG_LEVEL_DEBUG,
 		true,
 		serviceConfigs,
 		map[int]int{
-			NODE_SERVICE_ID: NODE_CONFIG_ID,
+			NODE_SERVICE_ID:           NODE_CONFIG_ID,
```

Comment:
So one thing I was thinking: what if we make this map a param to the function? As-is, this will mean that every test will now have a delegator node, using ID 1 (which will break the test I'm writing).

Reply:
Following up on this - I'd prefer not to have every network in this file get a delegator if we can avoid it